apps/services/sessions/src/app/cleanup/worker.ts

```diff
@@ -0,0 +1,21 @@
+import { NestFactory } from '@nestjs/core'
+
+import { SessionsCleanupWorkerModule } from './worker.module'
+import { SessionsCleanupService } from './worker.service'
+
+export const worker = async () => {
+  try {
+    console.log('Sessions cleanup worker started.')
```

Comment:
Use the `logger` instead of console.log?

Reply:
Fixed.